src/file/imp/unix.rs

```diff
@@ -98,32 +98,44 @@ fn create_unix(dir: &Path) -> io::Result<File> {
     })
 }
 
+#[cfg(not(unix))]
```

Comment:
This is a unix only module so this shouldn't be necessary.

Reply:
It looks like this module is also used for redox. Does `target_os == "redox"`` always imply `unix` ?

Reply:
Ah. I forgot about that... You're right. Can we say "cfg(redox)" and "cfg(not(redox))"?

Reply:
Yes, that would be clearer than `unix` and `not(unix)`.